tools/msvs/msi/nodemsi.wixproj

```diff
@@ -54,9 +54,12 @@
     </WixExtension>
   </ItemGroup>
   <ItemGroup>
-    <!-- <EmbeddedResource Include="i18n\de-de.wxl" /> -->
-    <EmbeddedResource Include="i18n\en-us.wxl" />
+    <!--
```

Comment:
@evanlucas yes, this is supposed to be commented for now. The translations are not yet active, the installer only has english.

@fhemberger is there a plan to release with translations? Are we just waiting for more translations to arrive? On the build side, we use the stable release of WiX, so english and german are the only languages supported. The WiX website recommends not to use the weekly versions in production, but we could perhaps add just the translation files to the release version, if it works.